pandas/io/parquet.py

```diff
@@ -208,12 +219,19 @@ def __init__(self):
             )
         self.api = fastparquet
 
-    def write(self, df, path, compression='snappy', index=None, **kwargs):
+    def write(self, df, path, compression='snappy', index=None,
+              partition_cols=None, **kwargs):
         self.validate_dataframe(df)
         # thriftpy/protocol/compact.py:339:
         # DeprecationWarning: tostring() is deprecated.
         # Use tobytes() instead.
 
+        if 'partition_on' in kwargs:
+            partition_cols = kwargs.pop('partition_on')
```

Comment:
Should check that `partition_cols` is None before doing this. We would want to raise on

```python
df.to_parquet(engine='fastparquet', partition_cols=['a', 'b'], partition_on=['b', 'c'])
```

Reply:
done